include/cuco/detail/open_addressing_impl.cuh

```diff
@@ -174,7 +175,7 @@ class open_addressing_impl {
       predicate_{pred},
       probing_scheme_{probing_scheme},
       storage_{make_window_extent<open_addressing_impl>(
-                 static_cast<size_type>(static_cast<double>(capacity) / desired_load_factor)),
+                 static_cast<size_type>(static_cast<double>(n) / desired_load_factor)),
```

Comment:
Do we need to `std::ceil` the result of the division?

Reply:
Done